src/blueprint/from_config/operators/call.rs

```diff
@@ -0,0 +1,82 @@
+use crate::blueprint::*;
+use crate::config;
+use crate::config::{Config, Field, GraphQLOperationType};
+use crate::directive::DirectiveCodec;
+use crate::try_fold::TryFold;
+use crate::valid::Valid;
+
+pub fn update_call(
+  operation_type: &GraphQLOperationType,
+) -> TryFold<'_, (&Config, &Field, &config::Type, &str), FieldDefinition, String> {
+  TryFold::<(&Config, &Field, &config::Type, &str), FieldDefinition, String>::new(
+    move |(config, field, type_of, name), b_field| {
+      let Some(call) = &field.call else {
+        return Valid::succeed(b_field);
+      };
+
+      if validate_field_has_resolver(name, field, &config.types).is_succeed() {
+        return Valid::fail(format!(
+          "@call directive is not allowed on field {} because it already has a resolver",
+          name
+        ));
+      }
+
+      let type_and_field = if let Some(mutation) = &call.mutation {
+        Valid::succeed(("Mutation", mutation.as_str()))
```

Comment:
theoretically, this works, but I didn't write a test as I did not find a practical way to use it.

If you can provide some examples I'll be happy to include them on automated testing @tusharmath

Reply:
I removed it from the implementation